[2025-09-22T00:36:17.979+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: processamento_silver.order_payments_to_silver manual__2025-09-22T00:32:12.586757+00:00 [queued]>
[2025-09-22T00:36:18.073+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: processamento_silver.order_payments_to_silver manual__2025-09-22T00:32:12.586757+00:00 [queued]>
[2025-09-22T00:36:18.078+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-09-22T00:36:18.201+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): order_payments_to_silver> on 2025-09-22 00:32:12.586757+00:00
[2025-09-22T00:36:18.254+0000] {standard_task_runner.py:60} INFO - Started process 932 to run task
[2025-09-22T00:36:18.305+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'processamento_silver', 'order_payments_to_silver', 'manual__2025-09-22T00:32:12.586757+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/processamento_silver.py', '--cfg-path', '/tmp/tmphxj01c7s']
[2025-09-22T00:36:18.327+0000] {standard_task_runner.py:88} INFO - Job 96: Subtask order_payments_to_silver
[2025-09-22T00:36:19.132+0000] {task_command.py:423} INFO - Running <TaskInstance: processamento_silver.order_payments_to_silver manual__2025-09-22T00:32:12.586757+00:00 [running]> on host e833383cbb8e
[2025-09-22T00:38:21.219+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='processamento_silver' AIRFLOW_CTX_TASK_ID='order_payments_to_silver' AIRFLOW_CTX_EXECUTION_DATE='2025-09-22T00:32:12.586757+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-22T00:32:12.586757+00:00'
[2025-09-22T00:36:20.038+0000] {sql.py:276} INFO - Executing: CREATE SCHEMA IF NOT EXISTS silver;

DROP TABLE IF EXISTS silver.olist_order_payments_dataset;

CREATE TABLE silver.olist_order_payments_dataset AS
SELECT
    order_id,
    payment_sequential::INTEGER,
    lower(trim(payment_type)) AS payment_type,
    payment_installments::INTEGER,
    CAST(payment_value AS NUMERIC(18,6)) AS payment_value
FROM bronze.olist_order_payments_dataset;
[2025-09-22T00:36:20.301+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 277, in execute
    hook = self.get_db_hook()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 188, in get_db_hook
    return self._hook
  File "/usr/local/lib/python3.8/functools.py", line 967, in __get__
    val = self.func(instance)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/common/sql/operators/sql.py", line 150, in _hook
    conn = BaseHook.get_connection(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/hooks/base.py", line 82, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/connection.py", line 479, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `***_db` isn't defined
[2025-09-22T00:36:20.432+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=processamento_silver, task_id=order_payments_to_silver, execution_date=20250922T003212, start_date=20250922T003617, end_date=20250922T003620
[2025-09-22T00:36:20.716+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 96 for task order_payments_to_silver (The conn_id `***_db` isn't defined; 932)
[2025-09-22T00:36:20.812+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-09-22T00:36:20.964+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
